### DISCUSSION TITLE
Sync history after initial connection

### DISCUSSION THREAD

**Discussion by BasselAshi** [Auties00/WhatsappWeb4j#421] [General]
After connecting for the second time and using the serialized store, don't we have outdated data such as chats and contacts? Is there a way to sync again at any point or at least during later connections?

**Comment by Auties00** [comment]
> After connecting for the second time and using the serialized store, don't we have outdated data such as chats and contacts? Is there a way to sync again at any point or at least during later connections?

No the data gets updated don't worry, you can't ask for another history sync
